plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/Utils.php

```diff
@@ -54,14 +62,44 @@ public static function add_quantity_stepper_classes( $quantity_html ) {
 		return $processor->get_updated_html();
 	}
 
+	/**
+	 * Get standardized quantity input arguments for WooCommerce quantity input.
+	 *
+	 * @param \WC_Product $product The product object.
+	 * @return array Arguments for woocommerce_quantity_input().
+	 */
+	public static function get_quantity_input_args( $product ) {
+		return array(
+			/**
+			 * Filter the minimum quantity value allowed for the product.
+			 *
+			 * @since 2.0.0
```

Comment:
The `@since` version is the version when this hook was introduced in this part of the codebase. So here it would be `10.1.0`.

```suggestion
			 * @since 10.1.0
```